### PR TITLE
Fix up the styles and categories for the notes and status notes

### PR DESCRIPTION
**Pull request by paddyduke**
Replacing the component/doc statuses with fewer, more useful ones.

I've completely dropped the 'ready for review' and 'in production' statuses because I don't think they're useful. In particular the 'in production' one feels implicit on any section without another status.

I've also tidied up the styles a bit and swapped out the old icons, which required FontAwesome to work, with some new, standard SVG icons.

screen shot 2018-02-02 at 14 25 17

**NB:** I recommend checking out the branch and having a look when you're reviewing. It's hard to get a feel for the additions from the code alone.

### CODE REVIEW

File: assets/scss/docs/components/_note.scss [hunk 1]
```diff
@@ -30,66 +28,53 @@
...
+.Note--development .Note-title {
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Cpath fill='%23007ABF' fill-rule='evenodd' d='M19.5 19h1a.5.5 0 0 1 .5.5V21h-6v-1.5a.5.5 0 0 1 .5-.5h1v-7h3v7zm-12 0h1a.5.5 0 0 1 .5.5V21H3v-1.5a.5.5 0 0 1 .5-.5h1v-7h3v7zm7.69-12l2.31 4h-3.268l-2.31-4h3.269zm2.31 0H21v4h-1.19L17.5 7zM9.696 7l-.062.036L11.923 11H8.809L6.5 7h3.196zM4.191 7L6.5 11H3V7h1.19zM16.5 4h3v2h-3V4zm-12 0h3v2h-3V4z'/%3E%3C/svg%3E");
```
Comment (paddyduke) [inline assets/scss/docs/components/_note.scss]: This is obviously not using an inline element for the icon, but I figure we can update this when we create a proper Note component.

File: assets/scss/docs/components/_note.scss [hunk 2]
```diff
@@ -4,22 +4,20 @@
 
 
 .Note {
-  padding: 20px;
+  padding: 16px;
```
Comment (summers1981FA) [inline assets/scss/docs/components/_note.scss]: Hey Paddy, I noticed that this starts before other rule-sets and wasn't sure if we were still going alphabetically?

File: assets/scss/docs/components/_note.scss [hunk 3]
```diff
@@ -4,22 +4,20 @@
...
+  margin-bottom: 8px;
+  color: #181a1b;
   background-position: left center;
   background-repeat: no-repeat;
+  background-size: 24px;
```
Comment (summers1981FA) [inline assets/scss/docs/components/_note.scss]: Just a similar point to above about whether this should be in alphabetical order or not going forward?

File: assets/scss/docs/components/_note.scss (hunk 1 above)
Comment (summers1981FA) [reply assets/scss/docs/components/_note.scss]: Agree with this comment.

**Comment by paddyduke** [comment]
@summers1981FA I've alphabetised the rules and also very slightly bumped up the size of the target icon to match the others.

**Comment by paddyduke** [comment]
@summers1981FA If you're happy with this I'll get it merged first thing on Monday.

**Pull request merged**